src/platforms/php/guides/laravel/other-versions/laravel5.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Laravel 5.x
+description: "Learn about using Sentry with Laravel 5.x."
+sidebar_order: 200
+---
+
+<Note>
+
+Laravel 5.x is supported until version 2.14.x of the `sentry/sentry-laravel` package.
+
+</Note>
+
+Laravel is supported using a native package: [sentry-laravel](https://github.com/getsentry/sentry-laravel).
+
+This guide is for Laravel 5.x. We also provide instructions for [the latest Laravel](/platforms/php/guides/laravel/) as well as [Lumen-specific instructions](/platforms/php/guides/laravel/other-versions/lumen/).
+
+
+## Install
+
+Install the `sentry/sentry-laravel` package:
+
+```bash
+composer require "sentry/sentry-laravel:^2.14"
+```
+
+If you're on Laravel 5.5 or higher, the package will be auto-discovered. Otherwise you will need to manually configure it in your `config/app.php`.
```

Comment:
```suggestion
If you're on Laravel 5.5 or higher, the package will be auto-discovered. Otherwise, you will need to manually configure it in your `config/app.php`.
```